web/electric.json

```diff
@@ -0,0 +1,5 @@
+{
+  "app": "glossy-party-1694",
```

Comment:
@balegas -- @icehaunter has provisioned the `items-example` app. We should use this for all the simple items table based examples. (It's a pretty slug and it means data syncs across platform when running the demos).

Reply:
cool. Will gladly use that :)